src/attiny_rtc_i2c.rs

```diff
@@ -727,4 +727,131 @@ impl SharedI2C {
     pub fn tell_attiny_to_power_down_rp2040(&mut self, delay: &mut Delay) -> Result<(), Error> {
         self.try_attiny_write_command(REG_RP2040_PI_POWER_CTRL, 0x02, delay)
     }
+
+    fn attiny_read_eeprom_command(&mut self, command: u8, payload: &mut [u8]) -> Result<(), Error> {
```

Comment:
Probably shouldn't be called `attiny_read_eeprom_command`, since no attiny is involved here?